TPP.Core/Commands/Definitions/BadgeCommands.cs

```diff
@@ -253,18 +267,51 @@ public async Task<CommandResult> Pokedex(CommandContext context)
                     ResponseTarget = ResponseTarget.WhisperIfLong
                 };
             }
+            else if (mode.Equals(PokedexModeKantoooooooo))
+            {
+                ImmutableSortedDictionary<PkmnSpecies, int> ownedPokemons = (await _badgeRepo.CountByUserPerSpecies(user.Id));
+                int userOwnedRegionCount = ownedPokemons.Count(ownedPokemon =>
+                    ownedPokemon.Key.GetGeneration() == Generation.Gen1);
+                return new CommandResult
+                {
+                    Response = isSelf
+                        ? $"You have collected {userOwnedRegionCount}/151 distinct KANTOOOOOOOOOOOOOOO OhMyDog badge(s)"
+                        : $"{user.Name} has collected {userOwnedRegionCount}/151 distinct KANTOOOOOOOOOOOOOOO OhMyDog badge(s)"
+                };
+            }
+            else if (_pokedexModeRegions.ContainsKey(mode.ToLower()))
+            {
+                Generation modeGeneration = _pokedexModeRegions[mode.ToLower()];
+
+                // Getting the total number of distinct Pokemon from a single generation could also be a function in class PokedexData,
+                //   by using "GenMaxIds", but I'm not sure how to get the total number of Fakemons
+                int totalRegionCount = _knownSpecies.Count(pokemon => pokemon.GetGeneration() == modeGeneration);
+                ImmutableSortedDictionary<PkmnSpecies, int> ownedPokemons = (await _badgeRepo.CountByUserPerSpecies(user.Id));
+                int userOwnedRegionCount = ownedPokemons.Count(ownedPokemon =>
+                    ownedPokemon.Key.GetGeneration() == modeGeneration);
+                return new CommandResult
+                {
+                    Response = isSelf
+                        ? $"You have collected {userOwnedRegionCount}/{totalRegionCount} "
+                          + $"distinct {System.Threading.Thread.CurrentThread.CurrentCulture.TextInfo.ToTitleCase(mode)} badge(s)"
+                        : $"{user.Name} has collected {userOwnedRegionCount}/{totalRegionCount} "
+                          + $"distinct {System.Threading.Thread.CurrentThread.CurrentCulture.TextInfo.ToTitleCase(mode)} badge(s)"
```

Comment:
does that really need to be qualified with `System.Threading.Thread.CurrentThread.CurrentCulture.TextInfo`? Also, is there a culture-independent way of doing this? Is just uppercasing the first letter sufficient? E.g. `mode[0].ToUpper() + mode[1..]`

Reply:
Well, just **TextInfo.ToTitleCase** doesn't work as stated here

https://docs.microsoft.com/en-us/dotnet/api/system.globalization.textinfo.totitlecase?view=net-5.0

> culture-independent
what does that mean. Is using CurrentCulture bad?